packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart

```diff
@@ -23,17 +23,17 @@ const arrowHeadSize = 8.0;
 const distanceToArrow = 1.0;
 const arrowStrokeWidth = 1.5;
 
-/// Minimum size for scaling the flex children widget properly
-const renderedMinWidth = 175.0;
-const renderedMinHeight = 150.0;
+/// Hardcoded sizes for scaling the flex children widget properly
+const minRenderWidth = 200.0;
+const minRenderHeight = 300.0;
+const defaultMaxRenderWidth = 300.0;
+const defaultMaxRenderHeight = 300.0;
 
 const widgetTitleMaxWidthPercentage = 0.75;
 
 /// Hardcoded arrow size respective to its cross axis (because it's unconstrained)
-const outerHeightArrowIndicatorSize = 24.0;
-const outerWidthArrowIndicatorSize = 24.0;
-const innerHeightArrowIndicatorSize = 16.0;
-const innerWidthArrowIndicatorSize = 16.0;
+const heightArrowIndicatorSize = 48.0;
```

Comment:
why are these 48 and 42 instead of 48x48 or 42x42?

Reply:
this is because the font is rotated in the height and need additional space to rendering it horizontally instead of rendering it vertically